pkgs/development/python-modules/univers/default.nix

```diff
@@ -0,0 +1,46 @@
+{ lib
+, fetchPypi
+, fetchpatch
+, buildPythonPackage
+, setuptools-scm
+, attrs
+, packaging
+, pyparsing
+, semantic-version
+, semver
+, black
+, commoncode
+, pytestCheckHook
+, saneyaml
+}: buildPythonPackage rec {
+  pname = "univers";
+  version = "30.7.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-yM0SDBpkiZEbaZ0ugjiMwwUFKqZGbmh1JNlv5qvPAYo=";
+  };
+
+  patches = [
+    # Make tests work when not using virtualenv, can be dropped with the next version
+    (fetchpatch {
```

Comment:
Please mark this patch with a link to the relevant PR, so that we know when it is not needed anymore.